geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -341,6 +357,15 @@ function Home({
 
     }, []);
 
+    useEffect(() => {
+        if (!disableFeatured) {
+            const margin = 24;
+            const size = 320;
+            const count = Math.floor((width || window.outerWidth) / (size + margin));
+            fetchFeaturedResources(undefined, count);
+        }
+    }, [disableFeatured]);
```

Comment:
Just tried and this has not effect on the pagination, the count is always 5 in my case.
I think we need to move this directly inside the FeaturedList list component. So please remove this useEffect and see my next comment